### PR TITLE
Adding support for `graphql@14`

### PR DESCRIPTION
**Pull request by jnwng**
Will open this against #210 — making sure that this is passing integration before spamming that branch.

`graphql@14` is a major breaking change because they [no longer support the same engines]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,12 +28,12 @@
...
     "chai": "^4.0.2",
-    "graphql": "^0.13.0",
+    "graphql": "^14.0.2",
     "mocha": "^3.4.1",
     "rollup": "^0.45.0",
     "test-all-versions": "^3.3.2"
   },
   "peerDependencies": {
-    "graphql": "^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0"
+    "graphql": "^14.0.0"
```
Comment (hwillson) [inline package.json]: Can we keep the existing peer deps and just OR in 14.x? I don't think we should force people to use `graphql` 14.x yet.

Comment (jnwng) [reply package.json]: this'd likely be a major version bump for `graphql-tag` itself, i think. the reason i'm bumping the major here is that `graphql@14` is removing support for node@4, which would be a major breaking change for `graphql-tag` itself which previously supported node@4. 

what do you think?

**Comment by jnwng** [comment]
yep @hwillson im planning on merging into #210, just wanted to run the CI here to make sure it would work properly.

it's worth noting that #196 has been merged but not released, because it turns out it's a major breaking change to TypeScript users as well, adding to my desire to bump the major

**Comment by hwillson** [comment]
Thanks for the update @jnwng. I was hoping we didn't have to force people to use `graphql@14`, but if #196 is a breaking change, then a major version bump sounds like a good idea to me. Thanks for looking into this!

**Pull request merged**